portfolio/src/main/java/com/google/sps/data/Fields.java

```diff
@@ -0,0 +1,9 @@
+package com.google.sps.data;
+
+// Stores field names as constants.
+// Prevents errors due to typos when writing a field name string.
+public class Fields {
```

Comment:
NIT: I would name this RequestParameters, since that's where it's being used.

Reply:
Whoops, I didn't see this before I merged. I think I'll have to change it in my next PR.